benchmarks/Kernels/Distance/GowerBench.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Rubix\ML\Benchmarks\Kernels\Distance;
+
+use Rubix\ML\Datasets\Generators\Blob;
+use Rubix\ML\Kernels\Distance\Gower;
+use Rubix\ML\Transformers\LambdaFunction;
+
+/**
+ * @Groups({"DistanceKernels"})
+ * @BeforeMethods({"setUp"})
+ */
+class GowerBench
+{
+    protected const NUM_SAMPLES = 10000;
+
+    /**
+     * @var list<list<float>>
+     */
+    protected $aSamples;
+
+    /**
+     * @var list<list<float>>
+     */
+    protected $bSamples;
+
+    /**
+     * @var \Rubix\ML\Kernels\Distance\Gower
+     */
+    protected $kernel;
+
+    public function setUp() : void
+    {
+        $generator = new Blob([0, 0, 0, 0, 0, 0, 0, 0], 5.0);
```

Comment:
I think these data should be normalized as well, since it's hard to know the range of these values. All we know is that they are centered at zero and have a standard deviation of 5.0.